reana_workflow_controller/workflow_run_manager.py

```diff
@@ -425,6 +421,28 @@ def _create_job_spec(self, name, command=None, image=None,
             secrets_store.get_file_secrets_volume_as_k8s_specs(),
         ]
 
+        if os.getenv('FLASK_ENV') == 'development':
+            code_volume_name = 'reana-code'
+            code_mount_path = '/code'
+            k8s_code_volume = client.V1Volume(
+                name=code_volume_name
+            )
+            k8s_code_volume.host_path = client.V1HostPathVolumeSource(
+                code_mount_path
+            )
+            spec.template.spec.volumes.append(k8s_code_volume)
+
+            for container in spec.template.spec.containers:
+                container.env.extend(current_app.config['DEBUG_ENV_VARS'])
+                sub_path = f'reana-{container.name}'
+                if container.name == 'workflow-engine':
```

Comment:
I am not very happy about having to put this `if` here... the other option I see is to change the name of the workflow engine containers from `workflow-engine` to `workflow-engine-{self.workflow.type_}`. The biggest disadvantage I see with that approach is that retrieving logs for the workflow engines will differ from each implementation, which if I think about it, it will annoy me more than thinking that there is an `if` here ☺️:
```diff
-$ kubectl logs batch-yadage-d1dbdb66-fbe9-41d9-9509-1fb7ef947f0f-pdph5 workflow-engine
+$ kubectl logs batch-yadage-d1dbdb66-fbe9-41d9-9509-1fb7ef947f0f-pdph5 workflow-engine-yadage
-$ kubectl logs batch-cwl-d1dbdb66-fbe9-41d9-9509-1fb7ef947f0f-pdph5 workflow-engine
+$ kubectl logs batch-cwl-d1dbdb66-fbe9-41d9-9509-1fb7ef947f0f-pdph5 workflow-engine-cwl
-$ kubectl logs batch-serial-d1dbdb66-fbe9-41d9-9509-1fb7ef947f0f-pdph5 workflow-engine
+$ kubectl logs batch-serial-d1dbdb66-fbe9-41d9-9509-1fb7ef947f0f-pdph5 workflow-engine-serial
```